profile/profile.js

```diff
@@ -50,7 +50,7 @@ WDN.initializePlugin('modal', [function() {
 						let ability = response.abilities[i];
 						let modifiers = ability.has ? `data-userAbilityId=${ability.userAbilityId} checked` : '';
 						let checkbox = $(`<input id=${i} type="checkbox" value=${ability.abilityId} ${modifiers} />`);
-						let editLabel = $(`<label for=${i}>${ability.name}</label>`);
+						let editLabel = $(`<label for=${i} class="dcf-pl-1">${ability.name}</label>`);
```

Comment:
Lol I'm so sorry you had to wade through my horrible JQuery/JS mess in this file.

Reply:
I'm still sorry.